docs/set_up_admin_tails.rst

```diff
@@ -131,8 +131,8 @@ signed with the release signing key:
 .. code:: sh
 
     cd ~/Persistent/securedrop/
-    git checkout 1.2.2
-    git tag -v 1.2.2
+    git checkout 1.4.0rc1
+    git tag -v 1.4.0rc1
```

Comment:
If you compare https://github.com/freedomofpress/securedrop/pull/5061/files you'll note how the docs were bumped from 1.1.0 to 1.2.0 during the 1.2.0 release, while other versions were bumped to 1.3.0rc1. 

The  problem with bumping them to 1.4.0rc1 here is that this particular page is end user documentation that is published to https://docs.securedrop.org/en/latest/set_up_admin_tails.html (which the user will see if they switch from the default `master` branch to `latest`, which is built from `develop`). We want that documentation to refer to a version that actually exists, in case a user tries to follow the docs.